Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute.py

```diff
@@ -253,9 +271,701 @@ def fetch_incidents(client):
     return incidents
 
 
+def parse_limit_and_offset_values(args: dict) -> None:
+    """
+    Parse the offset and limit parameters to integers.
+
+    Args:
+        args (dict): command arguments.
+
+    """
+    offset, limit = args.get("offset", "0"), args.get("limit", "20"),
+    offset, limit = arg_to_number(arg=offset, arg_name="offset"), arg_to_number(arg=limit, arg_name="limit")
+
+    if offset is not None and offset < 0:
+        raise ValueError(f"offset parameter {offset} is invalid, cannot be a negative number")
+
+    if limit is not None and (limit < 1 or limit > MAX_API_LIMIT):
+        raise ValueError(f"limit parameter '{limit}' is invalid, must be between 1-50")
+
+    args["offset"], args["limit"] = offset, limit
+
+
+def update_query_params_names(names: List[Tuple[str, str]], args: dict) -> None:
+    """
+    Update the query parameters names.
+
+    Args:
+        names (list): a list of old name and new names to replace.
+        args (dict): a dict to replace its old key names with new key names.
+    """
+    for old_name, new_name in names:
+        if old_name in args:
+            args[new_name] = args.pop(old_name)
+
+
+def parse_date_string_format(date_string: str, new_format: str = "%B %d, %Y %H:%M:%S %p") -> str:
+    """
+    Parses a date string format to a different date string format.
+
+    Args:
+        date_string (str): the date in string representation.
+        new_format (str): the new requested format for the date string.
+
+    Returns:
+        str: date as a new format, in case of a failure returns the original date string.
+    """
+    try:
+        return dateparser.parse(date_string=date_string).strftime(new_format)
+    except AttributeError:
+        return date_string
+
+
+def epochs_to_timestamp(epochs: int, date_format: str = "%B %d, %Y %H:%M:%S %p") -> str:
+    """
+    Converts epochs time representation to a new string date format.
+
+    Args:
+        epochs (int): time in epochs (seconds)
+        date_format (str): the desired format that the timestamp will be.
+
+    Returns:
+        str: timestamp in the new format, empty string in case of a failure
+    """
+    try:
+        return datetime.utcfromtimestamp(epochs).strftime(date_format)
+    except TypeError:
+        return ""
+
+
+def get_api_filtered_response(
```

Comment:
use this func just to do the list slicing, do not send the request from it

Reply:
ack